addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -135,7 +135,7 @@ function changePaddingSize(direction) {
     }
     return {
         trigger: `.oe_overlay.ui-draggable.o_we_overlay_sticky.oe_active .o_handle.${paddingDirection}`,
-        content: _t(`<b>Slide</b> this button to change the ${direction} padding`),
```

Comment:
@qsm-odoo FYI, after seeing this change I asked MOC if it was possible to add a check against template strings in translations on the runbot, which he made, we're waiting for this PR to be merged before we can fix the existing cases and add that test on the runbot, but hopefully this kind of issue should no longer slip by unnoticed

Reply:
@sdegueldre the [branch](https://github.com/odoo-dev/odoo/tree/master-lint-template-strings-moc) is ready, but I cannot merge it until all the template-strings are fixed. That's why I need your help as asked on discord 🤙🏻

Reply:
@d-fence Yes I understood that, I thought this PR would be be merged quickly and I could fix the other existing cases after that but maybe I can already make a PR with all changes not covered by this one. I'll look at doing that tomorrow then, thanks again for the test, I think it confuses a lot of people :)